relstorage/storage.py

```diff
@@ -1073,6 +1073,9 @@ def _abort(self):
         if self.blobhelper is not None:
             self.blobhelper.abort()
 
+    def afterCompletion(self):
+        self._rollback_load_connection()
```

Comment:
Won't that happen when a writing transaction completes (commit/abort)?

Manual testing doesn't show a session idle after a commit or abort.